src/test/kotlin/com/zopa/ktor/opentracing/KtorOpenTracingTest.kt

```diff
@@ -289,7 +290,7 @@ class KtorOpenTracingTest {
     }
 
     @Test
-    fun `Client passes trace context in headers`() {
+    fun `Client passes trace context in headers and tags uuids`() {
```

Comment:
this test is already very complex (not easy to assert on the client headers ) so can we maybe have this in a different test please